backend-opal/main.js

```diff
@@ -2,14 +2,17 @@ require('dotenv').config();
 const express = require('express');
 
 const server = express();
+const PORT = 3001;
```

Comment:
@Yuan-EPAM you should create a `.env` file and put the port there so you don't need to change the server.listen part

Reply:
you can add these steps to the readme, so others can remember it too